packages/client-twitter/src/plugins/SttTtsSpacesPlugin.ts

```diff
@@ -396,21 +396,32 @@ export class SttTtsPlugin implements Plugin {
      */
     private async handleUserMessage(
         userText: string,
-        userId: UUID
+        userId: string // This is the raw Twitter user ID like 'tw-1865462035586142208'
```

Comment:
Do we also want to change the type in `processAudio`

Reply:
https://github.com/worksgoodcompany/eliza/blob/862a2cc086abc4cc61d1e786b7a55675911f3886/packages/client-twitter/src/plugins/SttTtsSpacesPlugin.ts#L263